src/components/Toast/Toast.module.css

```diff
@@ -1,170 +1,32 @@
-@import '../../design-tokens/mixins.css';
-
-/*------------------------------------*\
-    # TOAST
-\*------------------------------------*/
-
-/**
- * 1) Message of information, success, caution, or warning to the user
```

Comment:
Do we wanna keep this docstring for consistency?

Reply:
Um of course, will re-add 😅